task_1.py

```diff
@@ -0,0 +1,14 @@
+"""
+Напишите программу, удаляющую из текста все слова, в которых присутствуют буквы «а», «б» и «в».
+Ввод: значение типа <str>
+Вывод: значение типа <str>
+autopep8 ON
+"""
+
+
+txt = 'Напишите абв напиабв програбвмму программу, удаляющую из \
+    этого абв текста все вабвс слова, содерабващие содержащие абв'
+
+del_a = [i for i in txt.split() if 'а' and 'б' and 'в' not in i]
```

Comment:
Константин условие работает неверно. `'а' and 'б'` всегда возвращает `'б'`  можете проверить в консоли. Для вашей реализации нужно сделать так:
```suggestion
del_a = [i for i in txt.split() if not ('а' in i and 'б' in i and 'в' in i)]
```